db/migrate/20161022053828_create_authorized_user.rb

```diff
@@ -0,0 +1,11 @@
+class CreateAuthorizedUser < ActiveRecord::Migration
```

Comment:
we generally try and avoid manipulating data in our migrations - maybe put this in seed file?

https://robots.thoughtbot.com/data-migrations-in-rails